.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -43,6 +43,9 @@ Providing this information upfront will facilitate a smoother review process. --
 
 ## Checklist
 - [ ] Please read our [Vector contributor resources](https://github.com/vectordotdev/vector/tree/master/docs#getting-started).
+- [ ] If this PR introduces changes to [LICENSE-3rdparty.csv](https://github.com/vectordotdev/vrl/blob/main/LICENSE-3rdparty.csv), please
+  run `dd-rust-license-tool write` and commit the changes. More details [here](https://crates.io/crates/dd-rust-license-tool).
```

Comment:
```suggestion
- [ ] If this PR introduces changes Vector dependencies (modifies `Cargo.lock`), please
  run `dd-rust-license-tool write` to regenerate the [license inventory](https://github.com/vectordotdev/vrl/blob/main/LICENSE-3rdparty.csv) and commit the changes (if any). More details [here](https://crates.io/crates/dd-rust-license-tool).
```

I think contributors might not realize whan that file should be changed. I think telling them to rerun it if `Cargo.lock` is modified might be easier to follow.